src/app/(sidebar)/smart-contracts/contract-explorer/components/ContractStorage.tsx

```diff
@@ -157,7 +188,25 @@ export const ContractStorage = ({
             ),
           },
           { value: capitalizeString(vh.durability) },
-          { value: formatNumber(vh.ttl) },
+          {
+            value: (
+              <div className="TtlBox">
+                <span
+                  className="TtlBox__value"
+                  {...(vh.expired ? { "data-style": "expired" } : {})}
+                >
+                  {formatNumber(vh.ttl)}
+                </span>
+                {vh.expired && vh.durability !== "temporary" ? (
+                  <Text size="sm" as="p">
+                    <Link onClick={() => handleRestore(vh.key, vh.durability)}>
+                      Restore
+                    </Link>
+                  </Text>
```

Comment:
We shouldn't need the wrapper `Text` element here. We can turn the above `span` into `div` or make `.TtlBox` box into a flexbox to stack the elements.